CHANGELOG.md

```diff
@@ -1,5 +1,139 @@
+    [issue/397]: https://github.com/cloudflare/wrangler/issues/397
+
+  - **Improve some error messages with human-readable suggestions - [jspspike], [issue/1157] [pull/1329]**
+
+    Previously, you would see not-super-helpful error messages if your API Token was expired or missing some permissions, didn't have Workers Unlimited enabled and tried to upload to KV, or tried to create a namespace that already existed. But we strive for helpful, informative error messages!
+
+    Now, you'll see the following error messages as appropriate:
+    ```
+    10026 => "You will need to enable Workers Unlimited for your account before you can use this feature.",
+    10014 => "Namespace already exists, try using a different namespace.",
+    10037 => "Edit your API Token to have correct permissions, or use the 'Edit Cloudflare Workers' API Token template.",
+    ```
+
+    [jspspike]: https://github.com/jspspike
+    [pull/1329]: https://github.com/cloudflare/wrangler/pull/1329
+    [issue/1157]: https://github.com/cloudflare/wrangler/issues/1157
+
+- ### Fixes
+
+  - **wrangler dev pretty-prints JSON messages from console.log() - [jspspike], [issue/1249] [pull/1371]**
+
+    Previously, when running `wrangler dev`, you could `console.log(JSON.stringify(some_object))`, but the output was hard to read: newlines weren't appropriately parsed and whitespace was a bit of a mess.
+
+    This change pretty-prints JSON received from the Inspector, so JSON output is much easier to read in `wrangler dev`.
+
+    [jspspike]: https://github.com/jspspike
+    [pull/1371]: https://github.com/cloudflare/wrangler/pull/1371
+    [issue/1249]: https://github.com/cloudflare/wrangler/issues/1249
+
+  - **Don't install wasm-pack when publishing a type:webpack project - [EverlastingBugstopper], [issue/745] [pull/1344]**
+
+    We shouldn't install wasm-pack if your project doesn't need it. We thought we fixed this in 1.7.0, but we didn't. This time it's fixed for real, we swear.
+
+    [EverlastingBugstopper]: https://github.com/EverlastingBugstopper
+    [pull/1344]: https://github.com/cloudflare/wrangler/pull/1344
+    [issue/745]: https://github.com/cloudflare/wrangler/issues/745
+
+  - **Pin wasm-pack and cargo-generate to specific versions instead of always downloading the latest version - [jspspike], [issue/1240] [pull/1358]**
+
+    Previously, every time `wasm-pack` or `cargo-generate` was needed, Wrangler would download and install the latest version. Now, we pin those dependencies to a specific version in every release of Wrangler. If you have either of these tools installed locally, Wrangler will use them if you have a more recent version.
+
+    [jspspike]: https://github.com/jspspike
+    [pull/1358]: https://github.com/cloudflare/wrangler/pull/1358
+    [issue/1240]: https://github.com/cloudflare/wrangler/issues/1240
+
+  - **Support batched KV PUT and DELETE operations to improve Workers Site publish performance - [jspspike], [issue/1191] [pull/1339]**
```

Comment:
mm - doesn't really improve performance. It just lets folks w/more than 10k static files use kv:bulk commands (and i think sites as well)

Reply:
Ooh, gotcha. Will update.